available_images.md

```diff
@@ -84,10 +84,10 @@ You can pin your version by adding the version tag to your URL as follows:
 
 | Framework         |Job Type 			     |Horovod Options 			       |CPU/GPU 	|Python Version Options     |  			 Example URL 			                                                                 | Latest Version|
 |-------------------|------------------------|---------------------------------|------------|---------------------------|----------------------------------------------------------------------------------------------------| --------------|
-|TensorFlow 2.3.0   |training 	             |Yes                        	   |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.3.0-cpu-py37-ubuntu18.04         |v1.0           |
-|TensorFlow 2.3.0   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.3.0-gpu-py37-cu102-ubuntu18.04   |v2.0           |
-|TensorFlow 2.3.0   |inference 	             |No                        	   |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.3.0-cpu-py37-ubuntu18.04         |v1.1           |
-|TensorFlow 2.3.0   |inference 	             |No                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.3.0-gpu-py37-cu102-ubuntu18.04   |v1.1           |
+|TensorFlow 2.3.1   |training 	             |Yes                        	   |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.3.0-cpu-py37-ubuntu18.04         |v1.0           |
+|TensorFlow 2.3.1   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.3.0-gpu-py37-cu102-ubuntu18.04   |v2.0           |
+|TensorFlow 2.3.1   |inference 	             |No                        	   |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.3.0-cpu-py37-ubuntu18.04         |v1.1           |
```

Comment:
Please change the URI to `763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.3.0-cpu-py37-ubuntu18.04` and version to `v1.5`.